content_schemas/formats/shared/definitions/_worldwide_office.jsonnet

```diff
@@ -0,0 +1,54 @@
+{
+  type: "object",
+  additionalProperties: false,
+  properties: {
+    access_and_opening_times: {
+      type: [
+        "string",
```

Comment:
I'm not sure I understand the benefit, but can these use the `body_html_and_govspeak` definition? That seems the most appropriate

Reply:
I'm not sure we could do that in this shared definition, as I think it'll affect the way we need to present the data from Whitehall, which will include modifying the existing worldwide offices.